src/main/java/com/mixpanel/android/mpmetrics/DecideChecker.java

```diff
@@ -263,6 +268,36 @@ private static int getDisplayWidth(final Display display) {
         }
     }
 
+    private static byte[] getUrls(RemoteService poster, Context context, String[] urls)
```

Comment:
I wanted to use the same code path for retrieving images (which are in a different package) - it seemed like performRequest was the right place to share it. In addition, getUrls wasn't used anywhere else, so this was an opportunity to reduce the public method count on the whole system.